components/script/dom/globalscope.rs

```diff
@@ -2158,15 +2158,20 @@ impl GlobalScope {
                 let ar = enter_realm(&*self);
 
                 let _aes = AutoEntryScript::new(self);
-                let options = CompileOptionsWrapper::new(*cx, filename.as_ptr(), line_number);
+                let options =
+                    unsafe { CompileOptionsWrapper::new(*cx, filename.as_ptr(), line_number) };
 
                 debug!("evaluating Dom string");
                 let result = unsafe {
-                    EvaluateUtf8(
+                    Evaluate2(
                         *cx,
                         options.ptr,
-                        code.as_ptr() as *const _,
-                        code.len() as libc::size_t,
+                        &mut SourceText {
+                            units_: code.as_ptr() as *const _,
```

Comment:
This makes me notice I forgot to file an issue to expose this function from rust-mozjs 👀 Let me file one!

https://github.com/servo/servo/blob/ca25e18791163de5c6af1267d876c7f5bb705f61/components/script/script_module.rs#L78-L85

Reply:
Ah, nah, I did file one at servo/rust-mozjs#490. Will send a PR to fix it and update here and `script_module`.